apigw/src/enduser/mapRoutes.ts

```diff
@@ -28,6 +29,43 @@ function createDigitransitProxy(path: string) {
           : {})
       }
       return proxyReqOpts
+    },
+    userResDecorator: (proxyRes, proxyResData) => {
+      function parseBody(): unknown {
+        if (!Buffer.isBuffer(proxyResData)) {
+          return undefined
+        }
+        const body = proxyResData as Buffer
```

Comment:
`Buffer.isBuffer()` is a type guard, so `as Buffer` should not be needed 🤔

Reply:
IDEA shows a warning, but TS is happy :thinking: